api/genotypes_loader.api.inc

```diff
@@ -48,41 +48,59 @@ function genotypes_loader_get_postgresql_version() {
  */
 
  /**
-  *  Give the different errors that are possible separate error codes. Also allow a parameter to be given to
+  *  @TODO: Give the different errors that are possible separate error codes. Also allow a parameter to be given to
   *  either "handle errors" or not- this way, if we see a certain error (such as not unique), we can handle it
   *  from outside of the function.
   *
-  *  Also - don't forget to refer to Lacey's new helper function on Github. This function needs to evolve
-  *  regardless! Whether or not we go with copying from a file remains to be seen...
   */
 
```

Comment:
This function docblock needs a lot more documentation ;-)... Please say what the function does and describe it's parameters. Also, make sure there's no empty line between the docblock and it's function.

Reply:
The docblock above this part describes the function, let me know if it needs more description. I moved the @TODO within the previous block and removed the empty line in commit https://github.com/UofS-Pulse-Binfo/genotypes_loader/commit/1277bb2ef06c2db55a61777187950a4e50ff4e10